homeassistant/core.py

```diff
@@ -524,30 +524,43 @@ def add_job(
         if target is None:
             raise ValueError("Don't call add_job with None")
         if asyncio.iscoroutine(target):
-            self.loop.call_soon_threadsafe(self.async_add_job, target)
+            self.loop.call_soon_threadsafe(
+                functools.partial(self.async_add_job, eager_start=True), target
```

Comment:
Should we put all args inside the partial?

Reply:
74717e98a2d33b710f578c00ae2c52508fbb7cb5